python-avd/pyavd/_eos_designs/structured_config/overlay/router_bgp.py

```diff
@@ -216,6 +216,30 @@ def _set_address_family_evpn(self: AvdStructuredConfigOverlayProtocol) -> None:
                     domain_remote=True,
                     activate=True,
                 )
+                if self.shared_utils.node_config.evpn_gateway.all_active_multihoming.enabled is True:
```

Comment:
Since the type is bool, and we are looking for True we can skip the `is True`
```suggestion
                if self.shared_utils.node_config.evpn_gateway.all_active_multihoming.enabled:
```